docs/adv/cli.rst

```diff
@@ -52,27 +55,56 @@ We can run this command for each student we would like to add. If you are comfor
     netsblox groups members MyFirstGroup
 
 Downloading Student Projects
-----------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 If we know the name of a project for a student that we would like to retrieve, we can download it locally using:
 
 .. code-block:: sh
 
     netsblox projects export PROJECT_NAME --user SOME_STUDENT_USERNAME
 
 Adding Starter Projects
------------------------
+~~~~~~~~~~~~~~~~~~~~~~~
+
 We also might want to set the accounts up with some starter projects. The following command will import a local project file to a student account:
 
 .. code-block:: sh
 
     netsblox projects import FILENAME --user SOME_STUDENT_USERNAME
 
 Resetting Passwords
--------------------
+~~~~~~~~~~~~~~~~~~~
+
 We can reset our password, or the password of a member of one of our groups, with:
 
 .. code-block:: sh
 
     netsblox users set-password NEW_PASSWORD --user SOME_STUDENT_USERNAME
 
 Omitting the "--user" option will reset your own password.
+
+Example Usage: Development
+--------------------------
+
+The CLI is useful for managing deployments and can be particularly useful when developing locally. It supports switching between multiple hosts easily and comes preloaded with the main NetsBlox cloud and local development (using the default configuration). A few examples are shown below.
+
+.. code-block:: sh
+
+    netsblox host list # list the hosts
+    netsblox host use dev # connect to another cloud (in this case, "dev")
```

Comment:
I think it would make sense to have a note saying that `dev` is the local development environment mentioned before, `use dev` has always been a bit confusing with dev.netsblox.org existing, even if `host list` will clarify.

Reply:
Good point. That certainly could be confusing. I will probably just rename "dev" to "local" in the CLI (ie, the default hosts) and update the docs accordingly.

Reply:
That works for me, don't forget to change it on  https://github.com/NetsBlox/NetsBlox/blob/master/README.md when you do.